tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -482,6 +478,10 @@ describe('scalar card', () => {
         selectors.getCurrentRouteRunSelection,
         new Map([['run1', true]])
       );
+      store.overrideSelector(
```

Comment:
I wish these could all go in to before each sections. However, I understand that the run is set here and it does make sense to do this after that.

Reply:
I definitely thought about it, but as you suspected I wanted it to mirror the run selection override.